src/Microsoft.ML.AutoML/API/AutoCatalog.cs

```diff
@@ -551,7 +551,7 @@ internal SweepableEstimator[] TextFeaturizer(string outputColumnName, string inp
         }
 
         /// <summary>
-        /// Create a list of <see cref="SweepableEstimator"/> for featurizing numeric columns.
+        /// Create a <see cref="SweepablePipeline"/> for featurizing numeric columns.
```

Comment:
>SweepablePipeline [](http://example.com/codeflow?start=32&length=17)

does `SweepableEstimator[]` is same as `SweepablePipeline`?

Reply:
Nope, and this comment is not reflecting the code. Thanks for catching that.

A bit more on `SweepablePipeline` and `SweepableEstimator`. you can consider `SweepablePipeline` as a pipeline in which estimators are all `SweepableEstimator`. Meanwhile, instead of a line structure, `SweepablePipeline` is tree-like structure, so in each pipe, it can have multiple estimators as well.